asab/api/service.py

```diff
@@ -23,6 +25,21 @@ def __init__(self, app, service_name="asab.ApiService"):
 		self.ZkContainer = None
 		self.AttentionRequired = {}  # dict of errors found.
 
+		path = asab.Config.get("general", "manifest_json_path")
+		if not os.path.isfile(path):
+			if os.path.isfile("/MANIFEST.json"):
+				path = "/MANIFEST.json"
+			elif os.path.isfile("MANIFEST.json"):
+				path = "MANIFEST.json"
+			else:
+				L.warning("MANIFEST.json file not found")
+		try:
```

Comment:
Manifest could be not provided (empty string) which is fine and should not generate warning